config/default.js

```diff
@@ -14,4 +22,13 @@ module.exports = {
     baseUrl: WEB_BASE_URL_LOCAL,
     port: WEB_PORT_LOCAL,
   },
+  mysql: {
+    hostUrl: HOSTNAME,
+    port: MYSQL_PORT,
+    user: MYSQL_USER,
+    password: MYSQL_PASSWORD,
+    charset: LANG,
+    dbUrl: DATABASE_URL,
+    dbName: MYSQL_DATABASE,
+  },
```

Comment:
여기에서 knex connection 구조에 맞춰주는거죠.

```suggestion
 mysql: {
    client: 'mysql2',
    connection: {
      host: 'localhost',
      port: 3306,
      user: 'root',
      password: 'root',
      database: 'hm_local',
      charset: 'utf8mb4',
      timezone: 'Z',
      decimalNumbers: true,
    },
    pool: {
      min: 0,
      max: 10,
    },
  },```

Reply:
그리고 default config는 사실상 local이라 민감한 내용이 없어서... 이참에 process.env 참조하지 않고 다 하드코딩으로 바꿔놓는건 어떨까요?